projectgen-micronaut/src/main/java/io/micronaut/starter/feature/acme/Acme.java

```diff
@@ -46,20 +43,8 @@ public String getDescription() {
         return "Adds support for ACME (Automated Certificate Management Environment)";
     }
 
-    @Override
-    public void apply(GeneratorContext generatorContext) {
-        generatorContext.getConfiguration().put("micronaut.server.ssl.enabled", "true");
-        generatorContext.getConfiguration().put("acme.enabled", "true");
-        generatorContext.getConfiguration().put("acme.tos-agree", "true");
```

Comment:
We cannot  delete configuration without providing it via a recipe.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-projectgen/pull/7/commits/e2d8d72b08ac9b01014390a49bba3c35d62556e1